modules/sb-start-asb.adoc

```diff
@@ -19,8 +19,20 @@ using the following procedure.
 // TODO: Change to *OpenShift Ansible Service Broker Operator* by GA (2 of 5)
 . Select the *Automation Broker Operator*.
 . Under *Provided APIs*, click *Create New* for *Automation Broker*.
-. Review the default YAML, set any additional {asb-name} configuration options,
-and click *Create*.
+. Add the following to the `spec` field in the default YAML provided:
```

Comment:
Don't need to add registry in spec, can use the default directly, by default, the registry is:
``` 
 - auth_name: asb-registry-auth
    auth_type: secret
    name: rhcc
    type: rhcc
    url: https://registry.redhat.io
    white_list:
    - .*-apb$
```

Reply:
this secret is created in last step.

Reply:
We should keep this to avoid confusion IMO.